tox.ini

```diff
@@ -19,12 +19,14 @@ deps =
     coverage
     pytest
     pytest-cov
-    pytest-django
     pytest-pythonpath
 
     py27: mock
-    django1.11: Django==1.11
```

Comment:
The travis build on master is now broken under django 1.7 because the latest version of pytest-django dropped support < 1.8.  This will fix it again